wiki/Website-Changelog.md

```diff
@@ -0,0 +1,172 @@
+# Changelog authoring best practices
+
+- [Tips for writing consistent changelog entries](#tips-for-writing-consistent-changelog-entries)
+- [Content structure](#content-structure)
+  - [Component changes](#component-changes)
+  - [New icons](#new-icons)
+  - [Breaking changes](#breaking-changes)
+- [Templates](#templates)
+  - [Templates for npm packages](#templates-for-npm-packages)
+  - [Templates for Figma changelog entries](#templates-for-figma-changelog-entries)
+
+---
+
+## Tips for writing consistent changelog entries
+
+- Changelog entries should always start with a paragraph (not a list).
+- Consider using a list instead of a long paragraph of text to communicate multiple changes for one element.
+- If you need to include a note (like a migration tip), add it to the bottom of the entry in a new paragraph. Consider using emphasized text, if you need to draw more attention to it.
+- Entries should use complete sentences but be short, clear, and descriptive enough to be helpful.
+- Use past tense.
+- Entries should always end with a period.
+- Use consistent terms for different types of changes: 
+  - Bugfix: “Fixed”
+  - New component, token, variant: “Added”
+  - Update: “Changed”, “Refactored”
+  - Removed or deleted: “Removed”
+  - Other edge cases: “Upgraded”, “Reduced”, “Prevented”
+- When referencing components, use the plain-text component name, e.g., `Accordion`.
+- All components or token names, including those in the summary, should be enclosed in backticks (`).
```

Comment:
```suggestion
**General Guidelines:**
- Start changelog entries with a paragraph, not a list.
- If multiple changes are made to a single element, consider listing them instead of creating a long text paragraph.
- If there's a need to include a note (e.g., a migration tip), place it at the bottom of the entry in a new paragraph. Emphasize the text to draw more attention if necessary.

**Writing Style:**
- Use complete sentences that are short, clear, and descriptive.
- Write in the past tense.
- End entries with a period.

**Terminology:**
- Utilize consistent terms for different types of changes:
  - Bugfix: "Fixed"
  - New additions: "Added" (e.g., new component, token, variant)
  - Updates: "Changed", "Refactored"
  - Removals: "Removed"
  - Other situations: "Upgraded," "Reduced," "Prevented."

**Component Referencing:**
- Reference components using their plain-text name, like `Accordion.`
- Enclose all component or token names, including those in the summary, in backticks (`).
```

Reply:
Slight suggestion for breaking up the content here

Reply:
@andgen404 I didn't make the change exactly as you proposed, but I made adjustments to use headings to break up the content better. Thanks for the suggestion!